include/wpe-fdo/view-backend-exportable.h

```diff
@@ -39,9 +39,21 @@ extern "C" {
 struct wl_resource;
 struct wpe_view_backend_exportable_fdo;
 
+struct wpe_view_backend_exportable_fdo_dmabuf_resource {
+    struct wl_resource* buffer_resource;
+    uint32_t width;
+    uint32_t height;
+    uint32_t format;
+    uint8_t n_planes;
+    int32_t fds[4];
```

Comment:
File descriptors should be of type `int`—which is the type used all around by the operating system for them. Is there some reason to use `int32_t` here instead?

Reply:
It's what's used in the linux_dmabuf_attributes struct:
https://github.com/Igalia/WPEBackend-fdo/blob/master/src/linux-dmabuf/linux-dmabuf.h#L13

Reply:
Fai enough :slightly_smiling_face: 

Let's merge this as-is for now, and I filed #56 as a cleanup to be done later on.

Reply:
But Weston's implementation of linux_dmabuf_attributes is `int`, it was us who converted it to `int32_t` when the linux-dmabuf code was adopted.